src/compass/utils/runconfig.py

```diff
@@ -139,11 +138,20 @@ def load_bursts(cfg: SimpleNamespace) -> list[Sentinel1BurstSlc]:
                 # does burst_id + pol exist?
                 burst_id_pol_exist = False
                 if burst_id in bursts:
-                    if any([True for b in bursts[burst_id] if b.pol == pol]):
+                    if any([True for b in bursts[burst_id]
+                            if b.polarization == pol]):
                         burst_id_pol_exist = True
 
+                # ok to add logic table
+                # is_ref\id_pol_exist | true  | false
+                # --------------------+-------+-------
+                # true                | false | true
+                # false               | true  | true
+                is_ref = cfg.input_file_group.reference_burst.is_reference
+                ok_to_add = not is_ref or not burst_id_pol_exist
```

Comment:
I'm not sure if I understand this logic. Could you please describe what we are doing here?
If a burst is reference, we don't add it to the list? Then how do we run rdr2geo?